src/cascadia/TerminalControl/TermControl.h

```diff
@@ -167,6 +167,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         winrt::Windows::UI::Xaml::Controls::SwapChainPanel::LayoutUpdated_revoker _layoutUpdatedRevoker;
 
+        std::optional<winrt::Windows::UI::Core::CoreCursor> _oldCursor; // when we toggle the cursor, we have to save it here to restore it
+        winrt::Windows::UI::Core::CoreCursor _textCursor;
+        winrt::Windows::UI::Core::CoreCursor _pointerCursor;

```

Comment:
(Correct me if I'm wrong) It looks like you only set these in the constructor. Why not just make them `constexpr` in the namespace?

Reply:
they're winrt objects. it sucks.